addons/mail/static/src/model/model_manager.js

```diff
@@ -997,12 +1031,26 @@ export class ModelManager {
      * @private
      * @param {Object} listener
      */
-    _markListenerToNotify(listener) {
+    _markListenerToNotify(listener, info) {
+        if (!(listener instanceof Listener)) {
+            console.error(listener);
+            throw new Error("Listener is not a listener");
```

Comment:
Isn't not critical to have this `console.error`, I'm ok with keeping the throw, lets just integrate the listener into it as string (and no big deal if its string value is not very useful).